R/bootstrap.R

```diff
@@ -1155,12 +1174,19 @@ dataTableOutput <- function(outputId) {
 #' )
 #' @export
 htmlOutput <- function(outputId, inline = FALSE,
-  container = if (inline) span else div, ...)
+  container = if (inline) span else div, fill = FALSE, fillItem = !inline, ...)
```

Comment:
I was envisioning `fillItem = FALSE` and `fill = fillItem`. (Or possibly vice versa.) Like, usually the contents of a `uiOutput` is not going to be a fill item, but if it is (and it matters) then probably the `uiOutput`'s container is a fill container?

Reply:
I don't like the fact that you'd have to remember to `uiOutput(fillItem = TRUE)` to opt-in to fill behavior, yet do `fill = FALSE` anywhere else to opt-out. I may have a better idea that I'll implement in the next commit.